bootstrap/src/main/config/mediaserver.xml

```diff
@@ -65,21 +65,7 @@
 	<dtls>
 		<minVersion>1.0</minVersion>
 		<maxVersion>1.2</maxVersion>
-		<cipherSuites>TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
-			TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
-			TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384,
-			TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,
-			TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
-			TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
-			TLS_DHE_RSA_WITH_AES_256_GCM_SHA384,
-			TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,
-			TLS_DHE_RSA_WITH_AES_256_CBC_SHA256,
-			TLS_DHE_RSA_WITH_AES_128_CBC_SHA256,
-			TLS_DHE_RSA_WITH_AES_256_CBC_SHA, TLS_DHE_RSA_WITH_AES_128_CBC_SHA,
-			TLS_RSA_WITH_AES_256_GCM_SHA384, TLS_RSA_WITH_AES_128_GCM_SHA256,
-			TLS_RSA_WITH_AES_256_CBC_SHA256, TLS_RSA_WITH_AES_128_CBC_SHA256,
-			TLS_RSA_WITH_AES_256_CBC_SHA, TLS_RSA_WITH_AES_128_CBC_SHA,
-			TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256</cipherSuites>
-		<certificate path="../conf/dtls/x509-server.pem" key="../conf/dtls/x509-server-key.pem" algorithm="rsa"/>
+		<cipherSuites>TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256</cipherSuites>
```

Comment:
Any reason to have restricted this? I would suggest to leave all possible values as default and let users filter.